web/src/todo_widget.ts

```diff
@@ -210,24 +212,123 @@ export class TaskData {
         },
     };
 
-    handle_event(sender_id, data) {
+    constructor({
```

Comment:
This movement of this code block is linter motivated. Data members (`handle` in this case) should be declared before any public constructor declarations.